src/Microsoft.Health.Fhir.Shared.Web/appsettings.json

```diff
@@ -24,7 +24,8 @@
         },
         "CoreFeatures": {
             "SupportsBatch": true,
-            "SupportsTransaction": true
+            "SupportsTransaction": true,
+            "BundleIncludeTotal": "Accurate"
```

Comment:
Should we have this off by default? And surface it in the template?

Reply:
Possibly, lets discuss this

Reply:
Defaulting this to "None" for now. It will take some additional work re-align our Crucible test suite expectations with a change to "Accurate"